### ISSUE TITLE
missing CHANGELOG.md

### ISSUE THREAD

**Issue by SeWieland** [oauth2-proxy/manifests#70]
I know there is a changelog in the actual oauth2-proxy repository, however I think it would be beneficial to have a changelog for the chart as well, since it is versioned separately.

**Comment by pierluigilenoci** [comment]
@SeWieland make sense

**Comment by pierluigilenoci** [comment]
@SeWieland do you want to open a PR for this?

**Comment by bluebrown** [comment]
There are two annotations fields in the chart.yaml which look interesting in this regard.
```yaml
annotations:
  properties:
    artifacthub.io/changes:
      description: |-
        This annotation is used to provide some details about the changes introduced by a given chart version. Artifact Hub can generate and display a ChangeLog based on the entries in the changes field in all your chart versions.
        This annotation can be provided using two different formats: using a plain list of strings with the description of the change or using a list of objects with some extra structured information (see example below). Please feel free to use the one that better suits your needs. The UI experience will be slightly different depending on the choice. When using the list of objects option the valid supported kinds are added, changed, deprecated, removed, fixed and security.
      type: string
    artifacthub.io/containsSecurityUpdates:
      description: Use this annotation to indicate that this chart version contains security updates. When a package release contains security updates, a special message will be displayed in the Artifact Hub UI as well as in the new release email notification.
      enum:
        - "true"
        - "false"
  type: object
...

**Comment by pierluigilenoci** [comment]
@SeWieland what about this idea?

**Comment by SeWieland** [comment]
Hi @pierluigilenoci, sorry for not responding!
I actually stopped using oauth2-proxy in our deployments - mainly due to https://github.com/oauth2-proxy/oauth2-proxy/issues/884... Due to limited time I'm currently not available for any PR on this issue.
In regards to artifacthub - I never really used it anyways as I prefer to retrieve information about charts at the location of their source code.
Thus I think that a proper changelog + versioning cannot be replaced by it, but sure - it would be helpful and a step in the right direction!

**Comment by pierluigilenoci** [comment]
@SeWieland thank you for your feedback.

**Comment by pierluigilenoci** [comment]
Closed with #92
